packages/pages-shared/__tests__/jest.setup.ts

```diff
@@ -1,3 +1,10 @@
-import makeServiceWorkerEnv from "service-worker-mock";
+import { fetch, Headers, Request, Response } from "@miniflare/core";
+import { HTMLRewriter } from "@miniflare/html-rewriter";
 
-Object.assign(globalThis, makeServiceWorkerEnv());
```

Comment:
I was getting issues with service-worker-mock not supporting `response.arrayBuffer()` which miniflare's in-memory cache implementation relies on. So I got rid of it!

Reply:
I should add a comment here explaining that we need to provide API's to the global scope as needed

Reply:
Maybe add a `miniflare-test` polyfill and use that here?


https://github.com/cloudflare/wrangler2/blob/main/packages/pages-shared/environment-polyfills/miniflare.ts